docs/mint/constructor.md

```diff
@@ -0,0 +1,33 @@
+
+Constructor
+===========
+  
+{% swagger method = "c0nstruct0r" path = " " baseUrl = " " summary = "constructor" %}  
+{% swagger-description %}  
+Initialize the contract  
```

Comment:
Maybe giving more information about what is initialized in the constructor or what this Smart Contract, throw this Constructor, allow to do. 

Because a comment "Initialize the contract" is not giving any real information. In addition to that, as the Constructor is the first "Object" when you navigate in the Gitbook, It should contain a quick description of what do or what is this Smart Contract.

What do you think ?

Reply:
I added constructor scope information.